spec/features/user_views_profile_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+feature 'user checks his profile page', %(
+  As a signed up user
+  I want to view my profile page
+  So that I can see my account information) do
+  scenario 'view account information' do
+    user = FactoryBot.create(:user)
+
+    visit user_path(user)
+
+    expect(page).to have_content(user.username)
+    expect(page).to have_content(user.email)
+    expect(page).to have_content(user.profile_photo)
+  end
```

Comment:
What if a user tries to log into a profile page that's not theirs?

Reply:
we plan to set this up once we have added admin functionality